File: android-app/baselibrary/src/main/java/com/musketeer/baselibrary/Activity/BaseUITask.java
package com.musketeer.baselibrary.Activity;

import android.os.Bundle;

/**
 * Created by zhongxuqi on 15-10-24.
 */
public interface BaseUITask {

    /**
     * 设置界面
     */
    void setContentView(Bundle savedInstanceState);

    /**
     * 初始化视图
     */
    void initView();

    /**
     * 初始化数据
     */
    void initEvent();

    /**
     * 初始化数据
     */
    void initData();
}

File: android-app/app/src/main/kotlin/com/musketeer/scratchpaper/activity/BaseUITask.kt
package com.musketeer.scratchpaper.activity

import android.os.Bundle

/**
 * Created by zhongxuqi on 15-10-24.
 */
interface BaseUITask {

    /**
     * 设置界面
     */
    fun setContentView(savedInstanceState: Bundle?)

    /**
     * 初始化视图
     */
    fun initView()

    /**
     * 初始化数据
     */
    fun initEvent()

    /**
     * 初始化数据
     */
    fun initData()
}
